opt_einsum/tests/test_contract.py

```diff
@@ -117,15 +117,14 @@ def test_drop_in_replacement(string):
     assert np.allclose(opt, np.einsum(string, *views))
 
 
-@pytest.mark.skipif(sys.version_info[0] < 3, reason='requires python3')
 @pytest.mark.parametrize("string", tests)
```

Comment:
Thanks, I think there are two more in this file. I think this is good to go after those are patched?

Reply:
Great, I believe this is good to go now. Looking forward to using this in Pyro.